index.html

```diff
@@ -738,54 +738,59 @@ <h2>Terminology</h2>
   <section>
     <h2>Identifier</h2>
     <p>
-This section describes the formal syntax for <a>DIDs</a> and <a>DID URLs</a>. The term
-"generic" is used to differentiate the syntax defined here from syntax defined
-by <em>specific</em> <a>DID methods</a> in their respective specifications.
+This section describes the formal syntax for <a>DIDs</a> and <a>DID URLs</a>.
+The term "generic" is used to differentiate the syntax defined here from syntax
+defined by <em>specific</em> <a>DID methods</a> in their respective
+specifications.
     </p>
 
     <section class="normative">
       <h3>DID Syntax</h3>
 
       <p>
-The generic <a>DID scheme</a> is a <a>URI</a> scheme conformant with [[!RFC3986]].
+The generic <a>DID scheme</a> is a <a>URI</a> scheme conformant with
+[[!RFC3986]]. The ABNF definition can be found below, which uses the syntax in
+[[!RFC5234]] and the corresponding definitions for <code>ALPHA</code> and
+<code>DIGIT</code>. All other rule names not defined in the ABNF below are
+defined in [[RFC3986]]. All Decentralized Identifiers MUST conform to the
```

Comment:
```suggestion
defined in [[RFC3986]]. All <a>DIDs</a> MUST conform to the
```

Seems odd to refer to them occasionally as Decentralized Identifiers.